cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2317,4 +2317,342 @@
     </description>
   </property>
 
+  <!-- Client retry configuration for different program types -->
```

Comment:
This will break the generation of the documentation for this table. This should not be here, as the only comments should be titles dividing sections.